File: src/Business/Services/TemplateGroupService.java
package Business.Services;

import Business.Services.Base.*;
import Common.DataAccess.*;
import Common.Models.*;
import Common.Services.*;

public class TemplateGroupService extends ReferenceParentEntityService<TemplateGroup, Template> implements ITemplateGroupService {
    public TemplateGroupService(
            IGlobalDataAccess globalDataAccess,
            ITemplateGroupDataAccess entityDataAccess,
            ITemplateDataAccess childEntityDataAccess) {
        super(globalDataAccess, entityDataAccess, childEntityDataAccess);
    }
}

File: backend/Business.Impl/Services/TemplateGroupService.cs
using Business.Contracts.Services;
using Business.Impl.Services.Base;
using Business.Models.Entities;
using DataAccess.Contracts;
using Shared.Contracts;

namespace Business.Impl.Services;

internal sealed class TemplateGroupService : GroupService<TemplateGroup, Template>, ITemplateGroupService
{
	public TemplateGroupService(ISharedContext sharedContext, IAppUnitOfWork unitOfWork)
		: base(sharedContext, unitOfWork, unitOfWork.TemplateGroupRepo, unitOfWork.TemplateRepo)
	{
	}
}
